src/middleware/middleware.js

```diff
@@ -19,6 +19,19 @@ function createCtxAndReqUUID (req, res, next) {
       repositories: new RepositoryFactory()
     }
 
+    const page = req.header(CONSTANTS.PAGINATOR_HEADERS.PAGE)
+    req.ctx.paginator = {} // set paginator in ctx only if the page value is supplied by the user in the req header
+
+    if (page) {
+      if (page >= CONSTANTS.PAGINATOR_PAGE) {
```

Comment:
I think I might have lost track, but wouldn't this always trigger if someone tried to get their second page? Because `CONSTANTS.PAGINATOR_PAGE` is a constant of 1?

Reply:
This is what I get for trying to review while trying to listen in a meeting. I looked at this condition and skipped to the else!